migrations/20160115130333-add-secureid.js

```diff
@@ -0,0 +1,10 @@
+var dbm = global.dbm || require('db-migrate');
+var type = dbm.dataType;
+
+exports.up = function (db, callback) {
+  db.addColumn('participants', 'secureid', {type: 'string'}, callback);
```

Comment:
The ```secureid``` is not unique. How do we deal with collisions, or is the probability of generating two exact ```secureid``` small enough?